imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -369,7 +369,6 @@ function cartToSalesOrder(cart) {
   const salesOrder = {
     companyCode: companyCode,
     type: "SalesOrder",
-    code: cart._id,
```

Comment:
That's correct. You **can** pass an ID if you want, but it's not necessary since we don't do anything with the document.